src/SprykerEco/Zed/Unzer/Communication/Plugin/CheckoutRestApi/UnzerCheckoutDataExpanderPlugin.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Unzer\Communication\Plugin\CheckoutRestApi;
+
+use Generated\Shared\Transfer\RestCheckoutDataTransfer;
+use Generated\Shared\Transfer\RestCheckoutRequestAttributesTransfer;
+use Spryker\Zed\CheckoutRestApiExtension\Dependency\Plugin\CheckoutDataExpanderPluginInterface;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+
+/**
+ * @method \SprykerEco\Zed\Unzer\Business\UnzerFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\Unzer\UnzerConfig getConfig()
+ * @method \SprykerEco\Zed\Unzer\Communication\UnzerCommunicationFactory getFactory()
+ */
+class UnzerCheckoutDataExpanderPlugin extends AbstractPlugin implements CheckoutDataExpanderPluginInterface
+{
+    /**
+     * {@inheritDoc}
+     * - Expands `RestCheckoutDataTransfer` quote with `UnzerCredentialsTransfer`.
```

Comment:
please expand the specification (use facade method specification as a reference)

Reply:
Resolved in consultation with Ilya Kubanov.